### PR TITLE
Enhancement/heritable encapsulation #692

### PR DESCRIPTION
**Pull request by trejas**
- [x] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](/docs/CONTRIBUTING.md)
- [x] ✅ I've added tests where applicable to test my new functionality.
DID NOT SEE THE HERITABLE TESTS. fixing now.

- [x] 📖 I've made sure that my contracts are well-documented.
- [x] 🎨 I've run the JS/Solidity linters (`npm run lint:all:fix`) and fixed any issues.

Enhancement #692 

---

# 🚀 Description
Added encapsulation to Heritable public variables.

### CODE REVIEW

File: contracts/ownership/Heritable.sol [hunk 1]
```diff
@@ -50,6 +50,22 @@ contract Heritable is Ownable {
     heir = newHeir;
   }
 
+  /**
+   * @dev Use these getter functions to access the internal variables in
+   * an inherited contract.
+   */
+  function getHeir() public view returns(address) {
```
Comment (frangio) [inline contracts/ownership/Heritable.sol]: As of #666 we've adopted the convention that private state variables with public getters follow the naming convention that

- state variable has a suffix underscore `_`, and
- getter is the name of the variable without suffix underscore.

In this case, for example, we'd have `address private heir_` and `function heir() public view`.

Can you please change the PR to follow this convention?

Comment (trejas) [reply contracts/ownership/Heritable.sol]: Will do.

File: package.json [hunk 2]
```diff
@@ -57,6 +57,7 @@
   },
   "dependencies": {
     "dotenv": "^4.0.0",
-    "ethjs-abi": "^0.2.1"
+    "ethjs-abi": "^0.2.1",
+    "lint": "^1.1.2"
```
[the remark this answers is not in the document]
Comment (trejas) [reply package.json]: You are correct. Fixed.

**Comment by frangio** [comment]
Thanks @trejas! 😄

**Pull request merged**